desktop/components/current_auctions/README.md

```diff
@@ -0,0 +1,60 @@
+# Current Auctions
+
+A React component that displays a bar of four current Auctions.
+
+## Usage
+
+If using within a React context:
+
+```javascript
+import { CurrentAuctions } from 'components/current_auctions'
+
+class Page extends Component {
+  render() {
+    return (
+      <CurrentAuctions
+        sales={sales}
+      />
+    )
+  }
+}
+```
+
+If used within a non-React context, `index.jsx` will automatically mount provided there is a default `#react-mount-current-auctions` somewhere present in the DOM:
```

Comment:
Out of curiosity, why not require a selector always? Then it wouldn't necessarily need documentation of the implicit behavior. The way we have it now, the template that defines the mount point is coupled to the `CurrentAuctions` component via the `#react-mount-current-auctions` token.